services/config_data_sources/json_project_data_source.rb

```diff
@@ -22,6 +22,24 @@ class Project
     def self.attribute_to_type_map
       return { :@repo_config => GitRepoConfig }
     end
+
+    def self.map_enumerable_type(enumerable_property_name: nil, current_json_object: nil)
+      if enumerable_property_name == :@job_triggers
+        type = job_trigger_hash["type"]
```

Comment:
what's the job_trigger_hash here? I can't seem to find where it's defined.

Reply:
Looks like a typo, I think that should be `current_json_object` instead of `job_trigger_hash`

Reply:
cool let me fix this

Reply:
typo! 😅